src/ui/comment/Comment.js

```diff
@@ -60,10 +59,11 @@ class Content extends React.PureComponent {
 }
 
 const Comment = ({postingId, comment, focused, isPermitted, connectedToHome}) => (
-    <div className={cx("comment", "entry", {"focused": focused})} data-moment={comment.moment}>
+    <div className={cx("comment", "entry", {"focused": focused, "single-emoji": comment.singleEmoji})}
+         data-moment={comment.moment}>
         {comment.deleting ?
             <CommentDeleting/>
-        :
+            :
```

Comment:
I know that automatic formatting moves this colon to the right, but it's logical position is on the conditional operator level.